src/components/CopyToClipboard/CopyToClipboard.ts

```diff
@@ -72,14 +79,21 @@ export class CopyToClipboard extends ResultAction {
         if (navigator && navigator.clipboard && navigator.clipboard.writeText) {
             try {
                 await navigator.clipboard.writeText(text);
-                this.element.querySelector<HTMLElement>('.coveo-caption-for-icon').innerText = l('CopyToClipboard_copied');
             } catch (err) {
                 this.logger.error('Copy to clipboard failed.', text, err);
                 this.copyToClipboardFallback(text);
             }
         } else {
             this.copyToClipboardFallback(text);
         }
+        this.setToolipText(this.options.successTooltip);
```

Comment:
Would remove the parameter, it's stored in `this.options`

Reply:
Nope the function is use twice with different param